DataFormats/SiPixelRawData/src/SiPixelErrorsSoA.cc

```diff
@@ -0,0 +1,6 @@
+#include "DataFormats/SiPixelRawData/interface/SiPixelErrorsSoA.h"
+
+#include <cassert>
+
+SiPixelErrorsSoA::SiPixelErrorsSoA(size_t nErrors, const SiPixelErrorCompact *error, const SiPixelFormatterErrors *err)
+    : error_(error, error + nErrors), formatterErrors_(err) {}
```

Comment:
Would it make sense to move the constructor inline and just drop the .cc file ?

Reply:
Good point, there is little reason for an out-of-line constructor